jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/coder/TimDecoderHelper.java

```diff
@@ -134,10 +137,11 @@ public OdeData decode(RxMsgFileParser fileParser, SerialId serialId) throws Exce
             timMetadata.setLogFileName(fileParser.getFilename());
             timMetadata.setReceivedMessageDetails(new OdeTimSpecificMetadata(
                   new OdeTimSpecificMetadataLocation(
-                        OssLatitude.genericLatitude(new Latitude(fileParser.getLatitude())).toString(),
-                        OssLongitude.genericLongitude(new Longitude(fileParser.getLongitude())).toString(),
-                        OssElevation.genericElevation(new Elevation(fileParser.getElevation())).toString(),
-                        OssSpeedOrVelocity.genericSpeed(new Speed(fileParser.getSpeed())).toString()),
+                        OssLatitude.genericLatitude(new Latitude(fileParser.getLocation().getLatitude())).toString(),
```

Comment:
Since we are moving away from OSS, we should try not to use the J2735 classes, meaning Latitude, Longitude, etc. If we are getting those values in J2735 format (i.e. Integer), then just pass integer to OssLatitude for the conversion instead of creating a Latitude object.